content/tutorials/manage-the-rippled-server/configuration/run-rippled-as-a-validator.md

```diff
@@ -1,74 +1,191 @@
 
-See [the `validator-keys-tool` GitHub repository](https://github.com/ripple/validator-keys-tool/blob/master/doc/validator-keys-tool-guide.md) for more information about managing validator keys.
+- Issuing timely votes
 
+    An ideal validator's votes arrive quickly and not after a consensus round has already finished. Meeting recommended `rippled` server [system requirements](system-requirements.html), which include having a fast internet connection, helps with this.
 
-## Public-Facing Server
+- Identified
 
-To protect a production validator from [DDoS](https://en.wikipedia.org/wiki/Denial-of-service_attack) attacks, you can use a stock `rippled` server as a proxy between the validator and the outside network.
+    An ideal validator has a clearly identified owner. Setting up [domain verification](#request-domain-verification) is a good start. Ideally, XRP Ledger network UNLs include validators operated by different owners in multiple legal jurisdictions and geographic areas. This reduces the chance that any localized events could interfere with the impartial operations of trusted validators.
 
-1. [Set up the `rippled` validator.](run-rippled-as-a-validator.html)
 
-2. [Set up one or more stock `rippled` servers.](install-rippled.html)
 
-3. Configure the validator and stock `rippled` servers to be [clustered](cluster-rippled-servers.html) with each other.
+## Set Up a Proxy to Help Protect Your Validator
 
-4. Make the following configuration changes to your validator:
-    * Copy the `[ips_fixed]` list and paste it under `[ips]`. These fields should contain only the IP addresses and ports of the public-facing rippled(s). The validator connects to only these peers.
-    * Change `[peer_private]` to `1` to prevent its IP address from being forwarded.
+To help protect a validator from [DDoS](https://en.wikipedia.org/wiki/Denial-of-service_attack) attacks, set up one or more stock `rippled` servers as proxies between the validator and inbound traffic. ***TODO: Stated correctly? I think inbound traffic is clearer than "outside network" - but is it accurate? Do we have any recommendations around the minimum number of proxies to set up? Is the primary purpose of providing more than one proxy that if one goes down, you don't lose your connection to the network? Also curious about how the validator's "view" of the network changes when proxies are in place -- what does inbound traffic to a proxy-less validator look like vs inbound traffic to a validator with two proxies, for example? I don't quite understand how the views could differ, if at all.***
```

Comment:
This is correct. You want to set peer_private to avoid incoming connections and then setup your own outgoing connections to a proxy (rippled server). We could also instruct validators they can connect to our public hubs here as well, these will ensure more peers and higher connectivity. @mDuo13 @jhaaaa 

In general a "connect to the network" section would be nice for validators.

Reply:
Ripple runs a few `rippled` servers for the specific purpose of creating reliable connection points in the XRP Ledger peer-to-peer network. Those are our "public hub servers" (public hubs for short)

Reply:
Thank you, Rome! 

Sorry I'm being a little thick here:

- Where can validator operators find a list of public hub servers? This is not about ripple.com validators in the Validator Registry, right?
- Now this feels like a really silly question - but what kind of connection are we talking about? How should they connect to these public hubs? This is not about clustering or proxying, correct?